pkgs/by-name/si/simple-binary-encoding/package.nix

```diff
@@ -0,0 +1,112 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gradle,
+  jre,
+  runtimeShell,
+
+  # Generating the usage instructions for script
+  htmlq,
+  html2text,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "simple-binary-encoding";
+  version = "1.36.0";
+
+  src = fetchFromGitHub {
+    owner = "aeron-io";
+    repo = "simple-binary-encoding";
+    tag = finalAttrs.version;
+    hash = "sha256-xqfsJq86qIFBG1542GDWNP0ayGaOzfIZ1KR0s0m8vf0=";
+  };
+
+  nativeBuildInputs = [
+    gradle
+    htmlq
+    html2text
+  ];
+
+  mitmCache = gradle.fetchDeps {
+    pname = "simple-binary-encoding";
+    pkg = finalAttrs.finalPackage;
+    data = ./deps.json;
+  };
+
+  gradleBuildTask = "build";
+
+  doCheck = true;
+  gradleCheckTask = "runJavaExamples";
+
+  outputs = [
+    "out"
+    "all"
```

Comment:
This is not evaluatable because the `.all` attribute is used for the list of all derivation outputs